frontend/src/libs/components/input/input.tsx

```diff
@@ -15,7 +15,7 @@ import styles from './styles.module.scss';
 type Properties<T extends FieldValues> = {
   control: Control<T, null>;
   errors: FieldErrors<T>;
-  label: string;
+  label?: string;
```

Comment:
```suggestion
  label: string;
```
use `isLabelVisuallyHidden` if you need to hide the label